src/api/ledger/parse/utils.js

```diff
@@ -4,6 +4,15 @@ const _ = require('lodash');
 const transactionParser = require('ripple-lib-transactionparser');
 const toTimestamp = require('../../../core/utils').toTimestamp;
 const utils = require('../utils');
+const BigNumber = require('bignumber.js');
+
+function adjustQualityForXRP(quality: string, takerGetsCurrency: string,
+    takerPaysCurrency: string) {
+  const shift = (takerGetsCurrency === 'XRP' ? 6 : 0)
```

Comment:
Yeah, this is just another way of saying that the `flipped` book is totally redundant, which is true.

Thinking about this makes me realize that we do have the option of merging the flipped book into the non-flipped book. The only disadvantage would be that some orders would appear differently than the way they were placed (they would have their direction flipped). I think it's probably better to merge because I don't think anyone really cares how the order was placed. I'll make a pull request.